<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>set_ansForm</class>
 <widget class="QWidget" name="set_ansForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>同步课堂</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame
{
	background-color:rgb(14 , 150 , 254);
	border-style: solid;
    border-width: 2px;
    border-color:rgb(14 , 150 , 254);
    border-radius: 20px;
}
QLabel
{
	
	color: rgb(255, 255, 255);
	
	font: 14pt &quot;Microsoft YaHei UI&quot;;
}
QWidget#widget
{
	
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>380</width>
     <height>607</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>380</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>380</width>
      <height>550</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>380</width>
      <height>550</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>375</width>
      <height>550</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox
{
	spacing: 5px;
	font: 500 14pt &quot;Microsoft JhengHei UI&quot;;
}
QCheckBox::indicator {
        width:30px;
        height: 30px;
        image: url(:/new/prefix1/Images/chk_no.png);
}
QCheckBox::indicator:enabled:checked {
        image: url(:/new/prefix1/Images/chk_yes.png);
}
QCheckBox::indicator:enabled:unchecked:pressed {

}</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>59</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>133</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="checkBoxA">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>选项A</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxB">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>选项B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxC">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>选项C</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxD">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>选项D</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxE">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>选项E</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBoxF">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>选项F</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="2">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>132</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>59</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="0" colspan="3">
      <widget class="QWidget" name="widget_2" native="true">
       <property name="styleSheet">
        <string notr="true">QWidget{	
	background-color: rgb(255, 255, 255);
	border-style: solid;
	border-top-color: rgb(200,200, 200);
	border-right-color: rgb(255, 255, 255);
	border-bottom-color: rgb(255, 255, 255);
	border-left-color: rgb(255, 255, 255);
    border-width: 2px;
}
QPushButton {
    border-style: solid;
    border-width: 2px;
    border-color:#e0f0ff;	
    border-radius: 20px;
	color:#297aff;	
	font: 400 20pt &quot;微软雅黑&quot;;
    background-color:#e0f0ff;	
    background-repeat: no-repeat;
    background-position: left center;
}
</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButton">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>45</height>
           </size>
          </property>
          <property name="text">
           <string>确定</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>129</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushButton_close">
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>45</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>100</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>147</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>请设定正确答案</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
